src/main/java/edu/metrostate/ics499/team2/services/RegisteredUserService.java

```diff
@@ -38,6 +38,12 @@ public RegisteredUser getByUserEmailAndPassword(String email, String password) {
 	}
 
 	public RegisteredUser create(@RequestBody final RegisteredUser registeredUser) {
-		return userRepo.save(registeredUser);
+		return isValid(registeredUser) ? userRepo.save(registeredUser) : null;
+	}
+	
+	@Override
+	public boolean isValid(RegisteredUser obj) {
```

Comment:
Did you happen to test this method? Wanted to make sure the logic to ensure uniqueness works in your use-case.